### ISSUE TITLE
Conky-nvidia constantly crashing and dumping core

### ISSUE THREAD

**Issue by rbieb** [brndnmtthws/conky#139]
Journalctl log:

```
Aug 27 14:14:06 StoraArch systemd-coredump[8174]: Process 1650 (conky) of user 1000     dumped core.
-- Subject: Process 1650 (conky) dumped core
-- Defined-By: systemd
-- Support: http://lists.freedesktop.org/mailman/listinfo/systemd-devel
-- Documentation: man:core(5)
-- 
-- Process 1650 (conky) crashed and dumped core.
-- 
-- This usually indicates a programming error in the crashing program and
-- should be reported to its vendor as a bug.
```

I'm running conky-nvidia 1.10.0-5 on Arch Linux 64 bit.

**Comment by Nooby4Ever** [comment]
Can you try [Conky-lua-nv](https://aur4.archlinux.org/packages/conky-lua-nv/) ? I'm also on arch and have no issues using nvidia with the linked package. It might be something specific to "your" system though.

As a work around you might want to try to use the solution i've mentioned in: #21 (which requires no nvidia support from conky so the build flag can be left out. I'm assuming you point to nvidia as being the problem)

***
If you meant it more of a general crash i think you need to provide more details.

**Comment by RoBi89** [comment]
I just downloaded conky-lua-nv and get the same error.

**Comment by lasers** [comment]
2 years 11 months passed. The log is generic and we don't see any core dumps here thus we can't analyze anything further about this issue. Thanks for the report. We need to fix #520 first too.

Superseded by https://github.com/brndnmtthws/conky/issues/520. Closing.
